lib/improver/tests/threshold/test_BasicThreshold.py

```diff
@@ -167,25 +167,15 @@ def setUp(self):
                              units='degrees')
```

Comment:
These latitude and longitude declarations should now be deleted as they are unused.
You can also delete the import of cf_units as this is no longer used following your changes.